src/templates/QuickstartDetails.js

```diff
@@ -164,13 +167,34 @@ const QuickstartDetails = ({ data, location }) => {
                 background-color: var(--divider-color);
               `}
             >
-              <div>
+              <div
+                css={css`
+                  display: flex;
+                  justify-content: space-between;
+                  @media (max-width: 1240px) {
+                    justify-content: flex-start;
+                  }
+                  @media (max-width: 760px) {
+                    flex-direction: column;
+                    align-items: stretch;
+                  }
+                `}
+              >
+                <InstallButton quickstart={pack} onClick={handleInstallClick} />
```

Comment:
These buttons look really far apart in the screenshots, can you look at the spacing we use on the Docs Site and adjust the margin v between the buttons so they are tighter? 

https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal/